extensions/exquery/restxq/src/main/java/org/exist/extensions/exquery/restxq/impl/RestXqTrigger.java

```diff
@@ -27,7 +27,9 @@
 package org.exist.extensions.exquery.restxq.impl;
 
 import java.util.List;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.LogManager;
```

Comment:
the double import of LogManager is in many files.... 
For the rest... it looks like a straight forward update.